src/file-picker/src/FilePicker.js

```diff
@@ -184,7 +188,20 @@ FilePicker.propTypes = {
    * Class name passed to the FilePicker.
    * Only use this if you know what you are doing.
    */
-  className: PropTypes.string
+  className: PropTypes.string,
+
+  /**
+   * Function that returns the call-to-action button text for selecting files.
```

Comment:
Additionally, can we add these to the `index.d.ts` file in the root of the repo under `FilePickerProps`? (or `FilePickerOwnProps` if that's where the inner props are defined)

Reply:
sry overlooked the ts typings